plone/app/vocabularies/terms.py

```diff
@@ -1,5 +1,5 @@
 # -*- coding: utf-8 -*-
-from binascii import b2a_qp
+from binascii import b2a_base64
```

Comment:
Please use `base64.urlsafe_b64encode()` instead:

- It doesn't add newlines, so no need to strip again.
- It produces tokens that can be embedded in URLs and filenames.